pubspec.lock

```diff
@@ -1,5 +1,5 @@
 # Generated by pub
```

Comment:
I think we can gitignore this file, as it's generated

Reply:
this file contains all packages version used.. if we don't commit this file, and a new incompatible version of a package is released, a new user may have problems